client/src/app.css

```diff
@@ -48,3 +48,17 @@ html, body {
 .flex-align-stretch { align-items: stretch; }
 .flex-1 { flex: 1 };
 .flex-0 { flex: 0 };
+
+
+.ng-scope, .app-container {
```

Comment:
take these out, or merge my most recent, found a more elegant way to handle them